Cargo.toml

```diff
@@ -105,10 +105,10 @@ uuid = { version = "1", features = [ "serde", "v4" ] }
 
 # git
 dropshot = { git = "https://github.com/oxidecomputer/dropshot", branch = "main", features = [ "usdt-probes" ] }
-omicron-common = { git = "https://github.com/oxidecomputer/omicron", branch = "main" }
+omicron-common = { git = "https://github.com/oxidecomputer/omicron", branch = "add-oximeter-producer-kind" }
```

Comment:
Note to self -- point to `main` once the corresponding Omicron PR merges.

Reply:
Ah, this pr: https://github.com/oxidecomputer/omicron/pull/4497

Reply:
Right. That PR introduces a new optional field, to avoid breaking builds right away. I'm adding the optional field to the out-of-tree repos now. Then I'll make that field required in Omicron, and then change the downstream consumers to follow.